_licenses/mpl-2.0.txt

```diff
@@ -3,7 +3,7 @@ title: Mozilla Public License 2.0
 redirect_from: /licenses/mozilla/
 source: https://www.mozilla.org/media/MPL/2.0/index.txt
 
-description: The Mozilla Public License (MPL 2.0) is maintained by the Mozilla foundation. This license attempts to be a compromise between the permissive BSD license and the reciprocal GPL license.
+description: Permissions of this weak copyleft license are conditioned on making available source code of licensed files and modifications of those files under the same license (or in certain cases, the GNU AGPLv3, GPLv2, LGPLv2.1, or later versions of those licenses) when distributed. Copyright and license notices must be preserved. Contributors provide an express grant of patent rights to users. However, a larger work using work may be distributed under different terms and without source code for other files the larger work.
```

Comment:
I very much agree with this rewording. (Previous wording did not carry at all the meaning of MPL 2.0.)

Maybe `(or in certain cases, the GNU AGPLv3, GPLv2, LGPLv2.1, or later versions of those licenses)` could be shortened. For instance, to `(or in certain cases, under a license of the GNU family)`.

The last sentence has some language mistakes. I would correct it to `However, a larger work using the licensed work may be distributed under different terms and without source code for the files added in the larger work.`